modules/custom-css/custom-css/js/core-customizer-css-preview.js

```diff
@@ -0,0 +1,47 @@
+// Originally based on https://raw.githubusercontent.com/xwp/wp-custom-scss-demo/master/custom-scss-demo-preview.js
+/* globals jpCustomizerCssPreview */
+(function( api, $ ) {
+	if ( api.settingPreviewHandlers ) {
```

Comment:
I'm not expecting much difficulty here, as this will only be enqueued if the new class is pulled in by the [`if ( function_exists( 'wp_get_custom_css' ) )` ](https://github.com/Automattic/jetpack/blob/80d4b437bcc6f87486394e63de308885053a26fc/modules/custom-css.php#L18)conditional.  I just left it there for anyone running trunk in an intermediate changeset.